### PR TITLE
Gcc 7.2.0 update

### PR DESCRIPTION
**Pull request by crystalct**

### CODE REVIEW

File: libs/Makefile [hunk 1]
```diff
@@ -10,12 +9,12 @@ all:
 	$(MAKE) -C spu_soundmodule
 
 install:
-	@echo "Install:"
 	$(MAKE) install -C spu_soundlib
 	$(MAKE) install -C libogg-1.2.1
 	$(MAKE) install -C libmpg123
 	$(MAKE) install -C audioplayerlib
 	$(MAKE) install -C libmodplay
+	$(MAKE) -C spu_soundmodule
```
Comment (wargio) [inline libs/Makefile]: ehm, this is not needed. you can add on the target the dependency on `all`

File: libs/audioplayerlib/audioplayer.c [hunk 2]
```diff
@@ -16,10 +16,10 @@
 
 */
 
-#include <sys/sem.h>
+
 #include "audioplayer.h"
 
-#include <vorbis/vorbisfile.h>
+#include "vorbis/vorbisfile.h"
```
Comment (wargio) [inline libs/audioplayerlib/audioplayer.c]: Why? this should be installed as a "system" library, so `<vorbis/vorbisfile.h>` should be available

**Review by wargio — CHANGES_REQUESTED** [review]

**Comment by crystalct** [comment]
Without  vorbis header:
crystal@DESKTOP-2FII39V ~/wargio/crystal_ps3soundlib/libs/audioplayerlib
$ make
[CC]  memfile.c
[CC]  audioplayer.c
audioplayer.c:57:5: error: unknown type name 'OggVorbis_File'
     OggVorbis_File vf;
     ^~~~~~~~~~~~~~
audioplayer.c:58:5: error: unknown type name 'vorbis_info'
     vorbis_info *vi;
     ^~~~~~~~~~~
audioplayer.c:112:37: error: unknown type name 'ogg_int64_t'; did you mean '__int64_t'?
 static int _fseek64_wrap(FILE * fp, ogg_int64_t off, int whence)
                                     ^~~~~~~~~~~
                                     __int64_t
audioplayer.c:117:26: error: unknown type name 'OggVorbis_File'
 int my_ov_open(FILE * f, OggVorbis_File * vf, const char *initial,

**Comment by wargio** [comment]
did you do `make all` and `make install` ?

**Comment by crystalct** [comment]
Sorry.... my mistake....
with 
#include <vorbis/vorbisfile.h>
works

**Comment by wargio** [comment]
np.

**Comment by wargio** [comment]
1 change is still missing.

**Comment by crystalct** [comment]
Damn CR+LR .... all file modified
make files for spu_soundmodule adjusted.....

**Pull request merged**